stable/traefik/values.yaml

```diff
@@ -64,6 +64,9 @@ forwardedHeaders:
   # - 10.0.0.0/8
 ssl:
   enabled: false
+  # Set secretName to use an externally defined secret
+  # must have keys "tls.crt", and "tls.key"
+  secretName: ""
```

Comment:
Can this be commented out by default?

Reply:
I commented this out and tested locally. check it out!